the-blue-alliance-ios/View Elements/Events/RankingTableViewCell.xib

```diff
@@ -25,40 +25,52 @@
                                 <rect key="frame" x="0.0" y="0.0" width="68" height="100.5"/>
```

Comment:
tbh I didn't know stack views existed until this project, they're amazing. They're the one thing I thought was missing from iOS ui creation that android has, turns out they weren't missing.